README.md

```diff
@@ -0,0 +1,9 @@
+# Spy Terminal
+
+The Spy Terminal is a command line program for encrypting secret messages to be decrypted using a shared secret password later
+
+## Getting Started
+
+1. Instatiate a virtual environment to isolated dependencies for this program: `virtualenv venv`
+2. Install all required dependencies: `pip install -r requirements.txt`
```

Comment:
A requirements file is a helpful way to document, and then (using the listed command) install all needed requirements for this program.